uber/tests/models/test_job.py

```diff
@@ -38,24 +38,43 @@ def session(request):
 class TestAssign:
     @pytest.fixture(autouse=True)
     def default_assignment(self, session):
```

Comment:
Any particular reason for this change?  The two versions are equivalent, but now there's a superfluous variable.  Though I suppose now that I look at it that by naming the superfluous variable ``error`` it's probably clearer to the reader what ``session.assign`` returns.  So I'm fine with it if you want to keep it this way, though I generally shudder at anything that "needlessly" adds a line of code :)